tests/console/zypper_info.pm

```diff
@@ -29,29 +39,35 @@ sub run() {
     my $expected_package_name_vim = 'Name *: vim';
     die "Missing package name. Expected: /$expected_package_name_vim/"
       unless $info_output_vim =~ /$expected_package_name_vim/;
+}
 
-    if (sle_version_at_least('12-SP3')) {
+sub run() {
+    select_console 'root-console';
+
+    # preparation of source repositories
+    my $cmd;
+    if (check_var('DISTRI', 'sle')) {
+        # SLE maintenance tests are assumed to be SCC registered
+        # and source repositories disabled by default
```

Comment:
No, there are two different srcpackage test here, one is using the src dvd and the other one is using SCC. src-dvd is not a temporary test until scc get tested. Both have to be tested.